aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -585,5 +585,34 @@ TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cpu) (
   }
 }
 
+Tensor& linalg_solve_sparse_csr_out(const Tensor& input, const Tensor& other, Tensor& result) {
+  #ifdef CUDART_VERSION
```

Comment:
I'm using it here, but when compiled with `USE_CUDA=1`, it still throws the error from the `#else` condition below.

Reply:
This file is not in `cuda/` folder and relevant CUDA headers are not included here so that `CUDART_VERSION` would be defined. Even though it's a CPU function, it uses a CUDA library and should be placed in `aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu`.